source/draw/RasterYuv422Semipacked.ooc

```diff
@@ -82,13 +86,10 @@ RasterYuv422Semipacked: class extends RasterPacked {
 		}
 	}
 	open: static func (filename: String) -> This {
-		x, y, imageComponents: Int
-		requiredComponents := 3
-		data := StbImage load(filename, x&, y&, imageComponents&, requiredComponents)
-		bgr := RasterBgr new(ByteBuffer new(data as UInt8*, x * y * requiredComponents), IntSize2D new(x, y))
-		result := This new(bgr)
+		bgr := RasterBgr open(filename)
```

Comment:
I just don't think it's obvious that `RasterYuv422Semipacked open` expects a BGR image. There is `openRaw` though, but this one seems more like the special case since it has a conversion. Anyway, it's not part of this PR so let's just leave it.